ppr-api/Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM python:3.8.6-buster
 USER root
```

Comment:
Add labels here?

Reply:
must have copy/pasted over it.
they should be there now.